workloads/fio/fio.py

```diff
@@ -0,0 +1,80 @@
+import logging
```

Comment:
Could you please add documentation to this file and explain that is being automatically consumed by Workload.setup()?

Reply:
Done, added documentation